### PR TITLE
Adds new reporter

### LINKED ISSUE #13405: Create new webcompatibility reporter
### Description
We need a new compatibility reporter to help users let us know when they are experiencing issues on certain pages.

### PR DESCRIPTION
**Pull request by NejcZdovc**
Auditors: @alexwykoff

Resolves #13405

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

## Test Plan:
- just run your browser

## Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by alexwykoff** [comment]
I'm checking this out and will update after seeing how it works. Thanks @NejcZdovc!

**Comment by alexwykoff** [comment]
It looks like the toggles aren't working. Also the submit button doesn't appear to be wired up.
Case 1: Toggles are not working
STR : 
1. build the branch
2. click on the toggles

Case 2: Submit button is not working
STR : 
1. build the branch
2. click on the submit button

File: app/extensions/brave/locales/en-US/app.properties [hunk 1]
```diff
@@ -202,6 +202,15 @@ removeBookmark=Remove bookmark
...
 removeFromReadingList=Remove from reading list
 replacedAds={{replacedAdsSize}} Ads replaced
+reporterTitle=Reporter
+reporterFooterButton=Send report
+reporterBraveVersion=Send Brave version
+reporterOSData=Send OS data
+reporterNotLoading=Page is not loading
+reporterClicksNotWorking=Click (action) is not working
+reporterPageCrashes=Page crashed
+reporterUrlLabel=Reported url
```
Comment (alexwykoff) [inline app/extensions/brave/locales/en-US/app.properties]: 'Reported url' maybe could be a little strange. Let's try 'URL'

**Comment by NejcZdovc** [comment]
this PR was created for video purpose only

**Comment by NejcZdovc** [comment]
will close it for now, we can always reopen it

**Pull request closed without merging**